R/block_cv.R

```diff
@@ -0,0 +1,160 @@
+#' Spatial block cross-validation
+#'
+#' Block cross-validation splits the area of your data into a number of
+#' grid cells, or "blocks", and then assigns all data into folds based on the
+#' blocks they fall into. By putting
+#'
+#' @details
+#' The grid blocks are drawn from can be controlled by passing arguments to
+#' [sf::st_make_grid()] via `...`. Some particularly useful arguments include:
+#'
+#' * `cellsize` Target cellsize, expressed as the "diameter" (shortest
```

Comment:
This section of documentation is to supplement the `sf` documentation; the entire `cellsize` description is:

```
cellsize: target cellsize
```

So I'm hoping to provide a bit more details on what exactly that represents here